src/changelog.txt

```diff
@@ -1,3 +1,11 @@
+---------------------------------------------------------------------------------------------------
+Version: 1.0.4
+Date: 30. 03. 2020
+  Changes:
+    - Updated for Factorio Version 0.18.0
+    - Technology updated to new science pack recipe names.
```

Comment:
I don't see the change that this references. Am I missing something?

Reply:
Oops the second line was supposed to be deleted.  Pushed it again haha